elftools/dwarf/callframe.py

```diff
@@ -323,6 +333,31 @@ def _read_augmentation_data(self, entry_structs):
             self.stream)['length']
         return self.stream.read(augmentation_data_length)
 
+    def _parse_lsda_pointer(self, structs, stream_offset, encoding):
+        """ Parse bytes to get an LSDA pointer.
+        """
+        assert encoding != DW_EH_encoding_flags['DW_EH_PE_omit']
+        basic_encoding = encoding & 0x0f
+        modifier = encoding & 0xf0
+
+        formats = self._eh_encoding_to_field(structs)
+
+        ptr = struct_parse(
+            Struct('Augmentation_Data',
+                   formats[basic_encoding]('LSDA_pointer')),
+            self.stream, stream_pos=stream_offset)['LSDA_pointer']
+
+        if modifier == 0:
```

Comment:
Please add a brief comment for what `modifier` means here

Reply:
Thanks for pointing out. I changed `0` to `DW_EH_encoding_flags['DW_EH_PE_absptr']`. I also added additional doc-string to describe where the implemented logic is coming from.